client/src/pages/UserLikes.jsx

```
@@ -100,17 +100,17 @@ export default function UserLikes() {
           </div>
         </div>
         <div className={styles.reviewWrapper}>
-          {!totalPage ? (
-            <div className={styles.noContentWrapper}>아직 좋아요한 내역이 없습니다.</div>
+          {isReview ? (
+            reviewCardLists.length !== 0 ? (
+              <ReviewCardLists reviewCardLists={reviewCardLists}></ReviewCardLists>
+            ) : (
+              <div className={styles.noContentWrapper}>아직 좋아요한 리뷰가 없습니다.</div>
+            )
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)